### ISSUE TITLE
We need a subduction cookbook

### ISSUE THREAD

**Issue by bangerth** [geodynamics/aspect#1828]
I must have been asked by half a dozen students today how to set up a subduction example. I don't think we have a cookbook on this, but I think it would be useful to show how to
* set up the initial temperature model
* set up the material models of the slabs and mantle
* etc

I don't think the example needs to be as complicated as some that have been done, for example by @anne-glerum , but it would be useful to discuss the *conceptual* approach of what one needs to address and how.

**Comment by naliboff** [comment]
Absolutely, this would definitely be useful for a lot of people. A student and I are currently working subduction models, so it is fresh on the mind and happy to help on this front. 

Perhaps a combination of a simplified cookbook (linear rheology) and a benchmark example (non-linear)? There are certainly tons of examples to choose from in the literature. 

@anne-glerum @MFraters - Does this sound like a reasonable starting point for most interested users?

**Comment by bangerth** [comment]
Yes, something simple is totally enough. The point is mostly to illustrate the WORKFLOW: how do you set up initial temperature fields, how do you set up material models, both of which should be faithful to some slab geometry. 

Having never done this, all I could think of is to describe the geometry of plared via a sequence of if-else statements, or via geomIO.

**Comment by anne-glerum** [comment]
Agreed!
At the moment I'm finishing up the repository with the input files and plugins that go with the paper on plasticity and subduction. These include:
- two 2D linear subduction benchmarks (Schmeling et al. 2008)
- two 2D and 3D viscoplastic subduction models with and without temperature- and strainrate dependence of viscosity 
Perhaps we can build on these for a cookbook?

**Comment by naliboff** [comment]
Agreed as well, the workflow for setting up the initial conditions really is the most important part for users new to modeling subduction (or modeling in general).

@anne-glerum - I completely forgot about the models associated with your paper. I imagine this will be enough for most users to get started and then a cookbook can be used to fill in any gaps or missing steps for a specific type of setup.

**Comment by bangerth** [comment]
Great, let's wait for @anne-glerum then. I'm happy to help with the writing of that section if that helps. As I mentioned before, it is not necessary to have something super realistic (though there is also nothing wrong with that). The main point should be to lay out what steps one has to take, and how one would go about that.

@anne-glerum -- let us know once you have something you'd like to share, and are comfortable sharing! Thanks in advance!

**Comment by bobmyhill** [comment]
We now have this in #5115
